modules/sr/robot/ruggeduino_devices.py

```diff
@@ -30,6 +34,10 @@ def read_value(self):
 
 
 class Microswitch:
+    """
+    A standard Webots touch sensor.
+    Reading from this sensor returns a boolean to match the SR API.
```

Comment:
Does the SR API have microswitches specifically? I wasn't aware of that.

Reply:
> Does the SR API have microswitches specifically?

Definitely doesn't.

Reply:
This module is a halfway point between Webots and the SR API. Perhaps the docstring was misleading by mentioning the SR API so I'll delete that part